src/plugins/navigation/review/review.js

```diff
@@ -447,6 +470,13 @@ export default pluginFactory({
         } else {
             this.flagItemButton.turnOff();
         }
+        if (this.customLayout) {
+            if (!this.explicitlyHidden) {
+                this.toggleButton.turnOn();
+            } else {
+                this.toggleButton.turnOff();
```

Comment:
Also add `this.toggleButton.turnOff();` to `disable()` method, for consistency with `flagItemButton`?

Reply:
Absolutely right. Good catch.